protocol-configuration/mainnetAddresses.ts

```diff
@@ -26,6 +26,7 @@ export const MainnetContractsConfig = {
     address: '0x956F47F50A910163D8BF957Cf5846D573E7f87CA',
     category: AddressCategory.Core
   },
+  // TODO: Deprecate once confident no dependents
```

Comment:
many more addresses in here would need to be marked deprecated.

Reply:
done what can be done so far, will update when can deprecate more